cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,12 +1,16 @@
 {% set base_metrics_memory = "200Mi" -%}
 {% set metrics_memory = base_metrics_memory -%}
+{% set base_metrics_cpu = "80m" -%}
+{% set metrics_cpu = base_metrics_cpu -%}
 {% set base_eventer_memory = "200Mi" -%}
 {% set eventer_memory = base_eventer_memory -%}
 {% set metrics_memory_per_node = 4 -%}
+{% set metrics_cpu_per_node = 1 -%}
 {% set eventer_memory_per_node = 500 -%}
 {% set num_nodes = pillar.get('num_nodes', -1) -%}
 {% if num_nodes >= 0 -%}
   {% set metrics_memory = (200 + num_nodes * metrics_memory_per_node)|string + "Mi" -%}
+  {% set metrics_cpu = (num_nodes + 80)|string + "m" -%}
```

Comment:
Actually, I have a PR in-flight to pod-nanny, that will enable passing less-than-millicore values to nanny and have them being correctly handled:
https://github.com/kubernetes/contrib/pull/1195

So once this is in, we should just change it to "0.5m" and we will be all happy, I believe.